ui/src/importer/importerTaskDetails/ImportedDocumentReport.js

```diff
@@ -17,6 +17,8 @@ class ImportedDocumentReportComponent extends Component {
       color = 'green';
     } else if (action === 'update' || action === 'replace') {
       color = 'yellow';
+    } else if (action === 'none') {
```

Comment:
action is none in case preview only? If so, why don't we set the action value to `preview`?

Reply:
because the preview is not an action in this context. None = nothing will be done with the record (f.e. when we are trying to delete a record, but the record does not exist in the db, the action will be none, either you run a preview or an actual import)

Reply:
Action is none if the document was not found in the dabatase, either in real or in preview delete.